cla_public/translations/cy/LC_MESSAGES/messages.po

```diff
@@ -4785,6 +4781,50 @@ msgstr "Ymddiriedolaeth Argyfyngau Cenedlaethol"
+msgstr "Yr Asiantaeth Cymorth Cyfreithiol (<abbr>LAA</abbr>) sy'n gyfrifol am y gwasanaeth Cyngor Cyfreithiol Sifil (<abbr>CLA</abbr>)."
+
+msgid "Feedback and contact information"
+msgstr "Adborth a gwybodaeth gyswllt"
+
+msgid "Compliance status"
+msgstr "Statws cydymffurfio"
+
+msgid "Time-out"
+msgstr "Terfyn amser"
+
+msgid "We are experiencing intermittent problems with the time-out feature where users are unexpectedly timed-out without the time-out warning appearing. We are currently investigating the source of this issue."
+msgstr "Rydym yn cael problemau ysbeidiol gyda'r nodwedd terfyn amser, lle bydd amser defnyddwyr yn dod i ben yn annisgwyl heb i'r rhybudd terfyn amser ymddangos. Rydym yn ymchwilio i'r broblem hon ar hyn o bryd."
+
+msgid "The ‘Hide this page’ button does not accurately describe the link destination. This fails WCAG 2.1 1.3.1 A (Info and relationships) and 2.4.6 AA (Headings and Labels)."
+msgstr "Nid yw'r botwm ‘Cuddio'r dudalen hon’ yn disgrifio cyrchfan y ddolen yn gywir. Mae hyn yn methu WCAG 2.1 1.3.1 A (Gwybodaeth a pherthnasoedd) a 2.4.6 AA (Penawdau a Labeli)."
+
+msgid "We understand that ‘Hide this page’ does not describe the link destination. We have purposely designed the feature to protect vulnerable users and provide a quick exit from the service rather than take users to a specific destination."
+msgstr "Rydym yn deall nad yw ‘Cuddio'r dudalen hon’ yn disgrifio cyrchfan y ddolen. Rydym wedi dylunio'r nodwedd yn fwriadol i ddiogelu defnyddwyr agored i niwed, a'u galluogi i adael y gwasanaeth yn gyflym yn hytrach nag arwain defnyddwyr i gyrchfan penodol."
+
+msgid "We are working to fix the time-out issue listed in this statement."
+msgstr "Rydym yn gweithio i ddatrys y broblem terfyn amser a restrir yn y datganiad hwn."
+
+msgid "Preparation of this accessibility statement"
+msgstr "Paratoi'r datganiad hygyrchedd hwn"
+
+msgid "This statement was prepared on %(day)s %(month)s %(year)s."
+msgstr "Paratowyd y datganiad hwn ar %(day)s %(month)s %(year)s."
+
+msgid "It was last reviewed on %(day)s %(month)s %(year)s."
+msgstr "Fe’i adolygwyd ddiwethaf ar %(day)s %(month)s %(year)s."
+
+msgid "This service was tested on %(day1)s %(month1)s %(year1)s and issues were re-tested on %(day2)s %(month2)s %(year2)s."
+msgstr "Profwyd y gwasanaeth hwn ar %(day1)s %(month1)s %(year1)s ac ailbrofwyd y materion a oedd angen sylw ar %(day2)s %(month2)s %(year2)s."
+
+msgid "The tests were carried out by the Digital Accessibility Centre (<abbr>DAC</abbr>)."
+msgstr "Cynhaliwyd y profion gan y Ganolfan Hygyrchedd Digidol (<abbr>DAC</abbr>)."
+
+msgid "The <abbr title='Digital Accessibility Centre'>DAC</abbr> tested core journeys, key functionality and interactive features that reflect the content that the average user may encounter."
+msgstr "Aeth y DAC ati i brofi teithiau craidd, swyddogaeth allweddol a nodweddion rhyngweithiol sy'n adlewyrchu'r cynnwys y gall y defnyddiwr cyffredin ddod ar eu traws."
```

Comment:
Should this have the abbreviation marked up and with a title attribute like the English does?

Reply:
Yes, good spot. Should have the abbreviation mark up like the original ENG text. Will make the change.